### DISCUSSION TITLE
Visual Studio 2022 (not VSCode) - allow user to use a key other than TAB to accept suggestions

### DISCUSSION THREAD

**Discussion by northernocean** [community/community#48735] [Copilot Conversations]
### Product Feedback

Using TAB to accept suggestions interferes with using TAB to enter tabs. This is jarring and unwelcome. It would be great if we could use a different key stroke (Shift-Tab, Alt-Tab, or -- preferred -- a user-defined keybinding). 

Note that this is different from "normal" Visual Studio 2022 intellisense completion because intellisense does not enter suggestions until I have started to type something. So I can always press TAB when I am on a new line and I will tab in.  But with Copilot, I can be on a new line and Copilot will start filling in code for me (wanted or unwanted) ... and now I can no longer use TAB as a normal tab key.

**Comment by MaxWilson** [comment]
Yes, PLEASE! The fact that Copilot is inserting code when I just want a new tab is driving me mad. I am writing F# but this would be an issue for any other whitespace-significant language such as Python. I may have to switch to VS Code simply to get Copilot to stop doing this. Copilot is valuable but please, PLEASE allow me to put it on some hotkey that isn't Tab.

**Reply by ayim** [reply]
@MaxWilson @zenerbufen there's actually an option to remap to the right arrow key here:
By checking this setting, it changes the behaviour for GitHub Copilot's code completions as well.

![image](https://github.com/community/community/assets/10951869/39c72ed4-c57f-4375-8b1e-8a87f5863ac3)
![image](https://github.com/community/community/assets/10951869/cd914e1c-df59-40fc-af72-d1d95a01dc08)

**Comment by zenerbufen** [comment]
+1

**Comment by internetfx** [comment]
Now the default hotkey CTRL+ALT+\ is gone in VS2022 with the integrated Copilot. This should definitely be added back and configurable.

**Reply by ayim** [reply]
Hey folks, there's a docs update pending.
You can use either `alt + ,` or `alt + ,` to trigger an inline suggestion if there isn't one already
These map to the `Edit.NextSuggestion` and `Edit.PreviousSuggestion` commands
